lib/logstash/plugin_mixins/jdbc/jdbc.rb

```diff
@@ -140,28 +140,18 @@ def jdbc_connect
 
     private
 
-    def load_drivers
-      return if @jdbc_driver_library.nil? || @jdbc_driver_library.empty?
-
-      driver_jars = @jdbc_driver_library.split(",")
-
-      # Needed for JDK 11 as the DriverManager has a different ClassLoader than Logstash
-      urls = java.net.URL[driver_jars.length].new
-
-      driver_jars.each_with_index do |driver, idx|
-        urls[idx] = java.io.File.new(driver).toURI().toURL()
-      end
-      ucl = java.net.URLClassLoader.new_instance(urls)
-      begin
-        klass = java.lang.Class.forName(@jdbc_driver_class.to_java(:string), true, ucl);
-      rescue Java::JavaLang::ClassNotFoundException => e
-        raise LogStash::Error, "Unable to find driver class via URLClassLoader in given driver jars: #{@jdbc_driver_class}"
-      end
-      begin
-        driver = klass.getConstructor().newInstance();
-        java.sql.DriverManager.register_driver(WrappedDriver.new(driver.to_java(java.sql.Driver)).to_java(java.sql.Driver))
-      rescue Java::JavaSql::SQLException => e
-        raise LogStash::Error, "Unable to register driver with java.sql.DriverManager using WrappedDriver: #{@jdbc_driver_class}"
+    def load_driver_jars
+      unless @jdbc_driver_library.nil? || @jdbc_driver_library.empty?
+        @jdbc_driver_library.split(",").each do |driver_jar|
+          begin
+            @logger.debug("loading #{driver_jar}")
+            # Use https://github.com/jruby/jruby/wiki/CallingJavaFromJRuby#from-jar-files to make classes from jar
+            # available
+            require driver_jar
+          rescue LoadError => e
+            raise LogStash::PluginLoadingError, "unable to load #{driver_jar} from :jdbc_driver_library, #{e.message}"
+          end
+        end
```

Comment:
can/should we also ensure that if `@jdbc_driver_class` was provided, it resolves after we've loaded the jars?

Reply:
It's effectively done [straight after](https://github.com/logstash-plugins/logstash-input-jdbc/blob/master/lib/logstash/plugin_mixins/jdbc/jdbc.rb#L178) this method is called. I'm not sure the extra check buys us a whole ton

Reply:
derp. you're right. you've got my `LGTM`